test/Microsoft.Health.Fhir.Shared.Tests.E2E.Common/FhirClient.cs

```diff
@@ -315,6 +315,26 @@ public async Task<FhirResponse<Bundle>> PostBundleAsync(Resource bundle)
             return await CreateResponseAsync<Bundle>(response);
         }
 
+        /**
```

Comment:
Weird format for comments, should just be `///` on each line

Reply:
This is interesting to me. /** is Java syntax for documentation comments. In the docs it says C# supports this syntax (as an equivalent to ///), but no one uses it. Switched to ///.